airqo_monitor/external/thingspeak.py

```diff
@@ -61,21 +65,19 @@ def get_data_for_channel(channel, start_time=None, end_time=None):
     return all_data
 
 
-def get_channel_ids_to_names():
-    """
-    Get all relevant channels to this tool (channels with AIRQO and without INACTIVE in the name)
-    """
+def get_all_channels():
```

Comment:
https://www.mathworks.com/help/thingspeak/listyourchannels.html maybe we can filter by tags here and explicitly only retrieve channels that correspond to the current tag?